src/main/java/org/apache/commons/collections4/SetUtils.java

```diff
@@ -149,15 +151,37 @@ public Iterator<E> createIterator() {
         };
     }
 
+    /**
+     * Returns a set of the difference of the given
+     * {@link Set}s, denoted by {@code setA \ setB} (or {@code setA - setB}).
+     * <p>
+     * The returned set contains all elements of {@code setA} that are not a member
+     * of {@code setB} at the moment of invocation. Later changes to {@code setA}
+     * or {@code setB} do not affect the returned set.
+     * Convenience method for difference(...).toSet().
+     * </p>
+     *
+     * @param <E> the generic type that is able to represent the types contained
+     *   in both input sets.
+     * @param setA  the set to subtract from, must not be null
+     * @param setB  the set to subtract, must not be null
+     * @return a new set with the difference of the given {@link Set}s
+     * @since 4.5.1
```

Comment:
Per semantic versioning, this would be for 4.6.0; 4.5.1 would be a bug fix release only.